NEWS.rst

```diff
@@ -53,6 +53,7 @@ possible, especially the following contributors:
 - Peter Cock
```

Comment:
I'd add a small blurb to the top of the file mentioning this change, something along the lines of: "PDBList now allows for the download of biological assemblies, in PDB and mmCIF format."